documentation/modules/uninstalling-mtv-cli.adoc

```diff
@@ -6,31 +6,48 @@
 [id="uninstalling-mtv-cli_{context}"]
 = Uninstalling {project-short} from the command line interface
 
-You can uninstall {project-first} from the command line interface (CLI) by deleting the +{namespace}+ project and the `forklift.konveyor.io` custom resource definitions (CRDs).
+You can uninstall {project-first} from the command line interface (CLI).
+
+[NOTE]
+====
+This action does not remove resources managed by the Operator, including custom resource definitions (CRDs) and custom resources (CRs). To remove these after uninstalling the Operator, you might need to manually delete the Operator CRDs.
+====
 
 .Prerequisites
 
 * You must be logged in as a user with `cluster-admin` privileges.
 
 .Procedure
 
-. Delete the project:
+. Delete the `forklift` controller by running the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ {oc} delete project {namespace}
+$ oc delete ForkliftController --all -n openshift-mtv
 ----
 
-. Delete the CRDs:
+. Delete the {project-short} Operator by running the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ {oc} get crd -o name | grep 'forklift' | xargs {oc} delete
+$ oc delete Subscription mtv-operator -n openshift-mtv
 ----
 
-. Delete the OAuthClient:
+. Delete the plugin console CR by running the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ {oc} delete oauthclient/forklift-ui
+$ oc delete ConsolePlugin forklift-console-plugin
 ----
+
+. Optional: Delete the custom resource definitions (CRDs) by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+{oc} get crd -o name | grep 'forklift.konveyor.io' | xargs {oc} delete
+----
++
+[NOTE]
+====
+Using this command ensures that you do not accidentally delete any KubeVert CDI resources when you delete the CRDs.
+====
```

Comment:
> Using this command ensures that you do not accidentally delete any KubeVert CDI resources when you delete the CRDs.

this explanation seems redundant and doesn’t add clarity to the actual purpose of the command. It’s evident that the command filters and deletes only the CRDs with the `forklift.konveyor.io` prefix, which inherently avoids touching any unrelated resources (like CDI resources).

To make the explanation more useful, I suggest clarifying the actual reason behind using this approach (e.g., safely cleaning up Forklift-specific CRDs without affecting other components). Focusing on why this specific command is needed, similar to the Note on line 11 above, rather than on what it doesn’t do, might provide more value to users.